AccountingSystem/AccountingSystem/CSVHandler.h

```diff
@@ -34,13 +35,22 @@ CSVHandler<T>::CSVHandler(std::string pathToFile, std::vector<std::string> colum
     : _pathToFile(pathToFile), _columnsName(columnsName), _toInputFormat(convertToInputFormat),
     _toOutputFormat(convertToOutputFormat) {}
 
+template<class T>
+inline CSVHandler<T>* CSVHandler<T>::GetHandler(std::string pathToFile, std::vector<std::string> columnsName, std::function<std::string(T)> convertToInputFormat, std::function<T(std::string)> convertToOutputFormat)
+{
+    std::ofstream myFile(pathToFile);
+    const auto delim = ' ';
+    for (auto i = 0; i < columnsName.size(); ++i)
+    {
+        myFile << columnsName[i] << delim;
+    }
+    myFile << "\n";
+    return new CSVHandler(pathToFile,columnsName,convertToInputFormat,convertToOutputFormat);
```

Comment:
Never return `new` via plain pointer. If you cannot avoid doing that, use smart pointers. But here that can be avoided
```suggesstion
auto handler = CSVHandler(pathToFile,columnsName,convertToInputFormat,convertToOutputFormat);
handler.init();
return handler;
```